Bottom: 4772bc50d4dec905d9d0a23c20bd7cb6696dbe1f
Top:    159b36f640c187856010320a7f040238bbfe6bee
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-19 01:34:16 +0000

Refresh of board-outline-polygon-generati

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 516e571..017eca6 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1520,6 +1520,7 @@ GhidDrawMask (int side, BoxType * screen)
   polygon.NoHoles = NULL;
   polygon.NoHolesValid = 0;
   polygon.BoundingBox = *screen;
+  polygon.Flags = NoFlags ();
   SET_FLAG (FULLPOLYFLAG, &polygon);
   common_fill_pcb_polygon (out->fgGC, &polygon, screen);
   poly_Free (&polygon.Clipped);
